tps-bench/src/main.rs

```diff
@@ -63,6 +65,11 @@ fn main() {
             if let Some(ref miner_config) = config.miner {
                 let rpc_urls = config.rpc_urls();
                 let miner = Miner::new(miner_config, rpc_urls);
+                let tip_block_number = net.get_confirmed_tip_number();
+                let cellbase_maturity_height = 1800 * *CELLBASE_MATURITY.lock().unwrap();
+                if tip_block_number < cellbase_maturity_height {
```

Comment:
Less Implicit behaviours.
If you really want it, I would suggest adding a miner option, like `ensure_height_greater_than = Some(18000)` (Default is `None`).

Reply:
The background of this pr is if matureds and unmatureds are both empty, tps-bench will not run until 1800 new blocks generated and UTXO not empty, because of we set miner generate 1 block/s, so this will block the process about 30min. I don't think is a implicit behavior, because it always will generate those blocks if we want to run tps-bench.

Reply:
Thanks, I see the background now.

But let me take another example to explain why I think a new miner option is better: Only 1 UTXO be matured is not enough for benching, benchmark needs many of matured UTXOs.
Hence providing a way to ensure the chain status before benching may be more general, such as `ensure_matured_capacity_greater_than = 100000`, `ensure_height_greater_than`.

Reply:
yeah, that should be much better that we have a config option like `ensure_matured_capacity_greater_than`, I'll add this into configuration file